Added files for the front end of the site.

Tomovasky opened the pull request:
Added files for the front end of the site.

Tomovasky committed:
Added files for the front end of the site.

kwhite17 commented on Civi_into.html:
```diff
@@ -0,0 +1,76 @@
+            </div>
+
+            <div class="form-group">
+                <label for="Prestinct" >Prestinct</label>
```
nit - spelling error: precinct

kwhite17 commented on event_page.html:
```diff
@@ -0,0 +1,91 @@
+            </div>
+        
+            <div class="form-group">
+                <label for="reason_stop" >What was the givin reason for the stop?</label>
```
nit: spelling - given

kwhite17 commented on event_page.html:
```diff
@@ -0,0 +1,91 @@
+            </div>
+
+            <div class="form-group">
+                <label for="reason_travel" >Why were you travling?</label>
```
nit: spelling - traveling

kwhite17 commented on event_page.html:
```diff
@@ -0,0 +1,91 @@
+            </div>
+
+            <div class="form-group">
+                <label for="road_cond" >What where the travel condtions like?</label>
```
nit: spelling - were

kwhite17 commented on event_page.html:
```diff
@@ -0,0 +1,91 @@
+        </form>
+        <div class="form-group d-flex justify-content-center">
+                
+            <button type="submit" class="btn btn-default">submit</button>
```
supernit: I think usually people would rock with `Submit` instead of `submit`

kwhite17 commented on index.html:
```diff
@@ -0,0 +1,83 @@
+        </div>
+    
+        <div id="about_the_project">
+            <p>Isum Copy... sdfskdhf;kjhsdjfh</p>    
+                <p>sdfjhk sdjfhksdhf</p> 
+                <p>skdkfjlsdkjf sdfkjsdl sdlfkj dirn sldkfj</p> 
+                <p>KDSjflskjd  twefkljs kjliji nfdsnl. </p>
+            </p>
```
We'll want to replace this with actual text. We don't want placeholder text like this on the website even if it's not fully polished yet.

Tomovasky replied:
Next time you are on the stream maybe you and @AndyM84 can talk about this one. 
As the two of you have differing points of view and it could be an interesting topic.

AndyM84 replied:
Well, we don't have differing points of view, as this is no longer a mockup 😉  Now that we're building it, yeah, you should ask us for the content and then plop it in.

For clarity, @kwhite17, I had said while we were doing mockups that they should avoid providing mockups with real copy in them since it has the chance to distract the discussion to the wrong things.

Tomovasky replied:
That makes sense now! I'm learning so much!

Tomovasky replied:
@AndyM84  and @kwhite17  What would you like me to put for the copy? ;)

AndyM84 replied:
For now, just put the title "Track the Blue" and then some lorem ipsum from something like https://www.loremipzum.com/en/

kwhite17 commented on index.html:
```diff
@@ -0,0 +1,83 @@
+            </div>
+
+            <div class="form-group">
+                <label for="Prestinct" >Prestinct</label>
```
nit: spelling - `Precinct`

kwhite17 commented on index.html:
```diff
@@ -0,0 +1,83 @@
+        
+        <div class="form-group d-flex justify-content-center">
+                
+            <a href="C:\Users\User\OneDrive\Desktop\Tracking the Blue\Civi_into.html"><button type="submit" class="btn btn-default">Your Info</button></a>
```
we're going to want the change the href to be a [relative path](https://stackoverflow.com/questions/24028561/relative-path-in-html#24028813) to the directory that we're in.

kwhite17 commented on index.html:
```diff
@@ -0,0 +1,83 @@
+
+<!-- Bootstrap CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+<link  rel="stylesheet"href=C:\Users\User\OneDrive\Desktop\tracking_the_blue\stylesheet.css>
```
ditto on [relative paths](https://stackoverflow.com/questions/24028561/relative-path-in-html#24028813)

kwhite17 reviewed (commented):
something that I missed out on is that we're going to want to be cognizant of [relative paths](https://stackoverflow.com/questions/24028561/relative-path-in-html#24028813). this is the same link to my other comments but it's important because the your links won't load on a remote server.

Tomovasky committed:
Updated after first way of PR.

Changed spelling errors and absolute paths to relative.

Change to copy on index  pending approved copy.
